source/_integrations/reolink.markdown

```diff
@@ -288,10 +288,12 @@ The following models have been tested and confirmed to work:
 - [Reolink Duo 3 PoE](https://reolink.com/product/reolink-duo-3-poe/)
 - Reolink Duo Floodlight ([PoE](https://reolink.com/product/reolink-duo-floodlight-poe/) and [Wi-Fi](https://reolink.com/product/reolink-duo-floodlight-wifi/))
 - Reolink TrackMix ([PoE](https://reolink.com/product/reolink-trackmix-poe/) and [Wi-Fi](https://reolink.com/product/reolink-trackmix-wifi/))
-- Reolink Video Doorbell ([PoE](https://reolink.com/product/reolink-video-doorbell/) and [Wi-Fi](https://reolink.com/product/reolink-video-doorbell-wifi/))
+- Reolink Video Doorbell ([PoE](https://reolink.com/product/reolink-video-doorbell/) and [Wi-Fi](https://reolink.com/product/reolink-video-doorbell-wifi/)) **
 
 *These models are discontinued and not sold anymore, they will continue to work with Home Assistant.
 
+**These models do not have HTTP/HTTPS ports open by default. You must use the Windows or Mac Client to enable one of them before configuring in Home Assistant.
```

Comment:
Let's remove this note here, and make it part of the instructions below instead.

Reply:
Removed.